contracts/Certificate.sol

```diff
@@ -162,6 +171,19 @@ contract Certificate is
     emit RegisterContractAddresses({removal: removal});
   }
 
+  /**
+   * @notice Used to decrement the discrepancy counter when removals are burned from this contract.
+   */
+  function decrementGuaranteeDiscrepancy(uint256 amount)
+    external
+    whenNotPaused
+  {
+    if (_msgSender() != address(_removal)) {
+      revert SenderNotRemovalContract();
+    }
```

Comment:
Could we revert if we decremented below 0? That way we're extra extra certain that we never replace more removals than necessary?

Reply:
as discussed IRL -- happening at the point of replacement in the followup PR